validation/exclusions_test.go

```diff
@@ -56,8 +56,37 @@ var exclusions = map[string]map[uint64]bool{
 		11763072: true, // sepolia-dev-0/base-devnet-0     No Public RPC declared
 	},
 	"Standard_Contract_Versions": {
-		11155421: true, // sepolia-dev-0/oplabs-devnet-0
-		11763072: true, // sepolia-dev-0/base-devnet-0
+		10:        true, // mainnet/op           MCP (at time of writing)
+		8453:      true, // mainnet/base         MCP (at time of writing)
+		957:       true, // mainnet/lyra         MCP (at time of writing)
+		1750:      true, // mainnet/metal        MCP (at time of writing)
+		34443:     true, // mainnet/mode         MCP (at time of writing)
+		291:       true, // mainnet/orderly      MCP (at time of writing)
+		424:       true, // mainnet/pgn          MCP (at time of writing)
+		7777777:   true, // mainnet/zora         MCP (at time of writing)
+		11155420:  true, // sepolia/op           MCP (at time of writing)
+		84532:     true, // sepolia/base         MCP (at time of writing)
```

Comment:
Those 2 have FPs, right?

Reply:
`sepolia/op` yes, `sepolia/base` I don't think so.

Reply:
If you check the `main` branch, you can see that `sepolia/base` and `mainnet/base` are both declared to be (only) on MCP contracts (and validated / not excluded as such).